hadoop-ozone/mini-cluster/pom.xml

```diff
@@ -25,6 +25,10 @@
   <name>Apache Ozone Mini Cluster</name>
   <description>Apache Ozone Mini Cluster for Integration Tests</description>
 
+  <properties>
```

Comment:
tests don't need javadoc, so this is fine.
Curious: does javadoc fail in this module?

Reply:
@jojochuang you have reported the failure in this module:

```
hadoop-ozone/mini-cluster/src/main/java/org/apache/hadoop/ozone/MiniOzoneHAClusterImpl.java:22: error: package org.apache.ozone.test.GenericTestUtils does not exist
[ERROR] import static org.apache.ozone.test.GenericTestUtils.PortAllocator.getFreePort;
...
```